reflex/utils/build.py

```diff
@@ -171,10 +203,12 @@ def export(
 
             checkpoints.extend(["Loading next-sitemap", "Generation completed"])
 
+        workdir = constants.Dirs.WEB_EXPORT if zip else constants.Dirs.WEB
```

Comment:
not clear why the directory depends on the zip flag; as a user i would assume `reflex export --no-zip` works just like `reflex export`, but does not produce the final zip file

Reply:
Because `reflex run --env prod` use the export function to build the static...

I didn't consider the --no-zip flag, but in that case we'll have to rewrite that whole part of the code.